modules/articles/tests/client/articles.client.controller.tests.js

```diff
@@ -1,14 +1,14 @@
-'use strict';
-
 (function () {
+  'use strict';
+
   // Articles Controller Spec
   describe('Articles Controller Tests', function () {
     // Initialize global variables
     var ArticlesController,
       scope,
+      rootScope,
```

Comment:
@trainerbill I was referring to these variable changes. They any variable that has a $ in the client side code also needs to have it in the tests for consistency.